src/components/LoanCard.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {useContext, useEffect} from 'react';
+import {Text, TouchableOpacity, View} from 'react-native';
+import {LoanCardPropType} from '../screens/CrowdfundingScreen';
+import AntDesignIcon from 'react-native-vector-icons/AntDesign';
+import LoanCardField from './LoanCardField';
+import BlueButton from './BlueButton';
+import {styles} from '../styles/ComponentStyle';
+import {LoginContext} from './Main';
+
+interface PropsType {
+  item: LoanCardPropType;
+}
+
+const convertNumToString = (num: number) => {
```

Comment:
@deottamprasad No need to write above function to convert number to string.
You can user javascript predefine function
`toString()`